diff --git a/dist/plugin/js/farm-to-vite-context.js b/dist/plugin/js/farm-to-vite-context.js
index 689b493f6539e6d260766584f20fc7097e1eb88c..600bd7ada8ba715c56789822f38712958a225b73 100644
--- a/dist/plugin/js/farm-to-vite-context.js
+++ b/dist/plugin/js/farm-to-vite-context.js
@@ -86,7 +86,7 @@ export function farmContextToViteContext(farmContext, currentHandlingFile, plugi
         parse: (_) => {
             throw new Error(`Vite plugin ${pluginName} is not compatible with Farm for now. Because parse(called by hook ${pluginName}.${hookName}) is not supported in Farm`);
         },
-        resolve: async (source, importer, options) => {
+        resolve: async (source, importer, options = {}) => {
             if (options.custom?.caller === `${pluginName}.${hookName}`) {
                 return null;
             }
